e2e/staticpvc.go

```diff
@@ -53,6 +63,56 @@ func getStaticPV(
 	return pv
 }
 
+func getMigrationStaticPV(
+	name, volHandle, size, secretName, secretNS, sc, driverName string,
+	blockPV bool,
+	options map[string]string) *v1.PersistentVolume {
+	pv := &v1.PersistentVolume{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: v1.PersistentVolumeSpec{
+			PersistentVolumeReclaimPolicy: v1.PersistentVolumeReclaimDelete,
+			Capacity: v1.ResourceList{
+				v1.ResourceStorage: resource.MustParse(size),
+			},
+			PersistentVolumeSource: v1.PersistentVolumeSource{
+				CSI: &v1.CSIPersistentVolumeSource{
+					Driver:           driverName,
+					VolumeHandle:     volHandle,
+					ReadOnly:         false,
+					VolumeAttributes: options,
+					NodeStageSecretRef: &v1.SecretReference{
+						Name:      secretName,
+						Namespace: secretNS,
+					},
+					ControllerExpandSecretRef: &v1.SecretReference{
+						Name:      secretName,
+						Namespace: secretNS,
+					},
+				},
+			},
+			StorageClassName: sc,
+			AccessModes:      []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce},
+		},
+	}
+
+	if blockPV {
+		volumeMode := v1.PersistentVolumeBlock
+		pv.Spec.VolumeMode = &volumeMode
+	} else {
+		volumeMode := v1.PersistentVolumeFilesystem
+		pv.Spec.VolumeMode = &volumeMode
+	}
+	if pv.Annotations == nil {
```

Comment:
This check is not required here as pv is a local object.

Reply:
Done